File: C/2423.c
// Problema: 2423 - Receita de Bolo | Resposta: Accepted
// Linguagem: C99 [+0s]             | Tempo: 0.000s

#include <stdio.h>

int main() {
    int x, o, c;
    scanf("%d %d %d", &x, &o, &c);
    int bolos[3] = {x/2, o/3, c/5};
    int minB = bolos[0];
    for (int i = 1; i < 3; i++) {
        if (bolos[i] < minB) {
            minB = bolos[i];
        }
    }
    printf("%d\n", minB);
    return 0;
}

File: PHP/2423.php
<?php
// Problema: 2423 - Receita de Bolo | Resposta: Accepted
// Linguagem: PHP (8.1.2) [+1s]     | Tempo: 0.120s

[$x, $o, $c] = array_map('intval', explode(' ', trim(fgets(STDIN))));
$bolos = [intdiv($x, 2), intdiv($o, 3), intdiv($c, 5)];
echo min($bolos) . PHP_EOL;
?>